packages/nc-gui/composables/useSharedView.ts

```diff
@@ -58,7 +60,7 @@ export function useSharedView() {
     const pageSize = paginationData.value.pageSize || 25
 
     const { data } = await $api.public.dataList(
-      sharedView?.value?.uuid,
+      (sharedView.value as any)?.uuid,
```

Comment:
Could it be `as ViewType`?

Reply:
Nope because ViewType is missing `uuid` otherwise it would work without the type hack :(

Reply:
Yes it have `uuid`

```ts
export interface ViewType {
  id?: string;
  title: string;
  deleted?: boolean;
  order?: number;
  fk_model_id?: string;
  slug?: string;
  uuid?: string;
  show_system_fields?: boolean;
  lock_type?: 'collaborative' | 'locked' | 'personal';
  type?: number;
  view?: FormType | GridType | GalleryType;
}
```

Rebuild `nocodb-sdk` and check it out.

Reply:
Ah okay, I missed the update then, will revert on those changes ^^